api/Element.json

```diff
@@ -2289,7 +2289,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": true
+              "version_added": "9"
```

Comment:
Do you mean you also tested with `attachEvent`?

Whatever the case may be, can you make the data for "oncopy" and "copy_event" entries make sense as a whole? Since the `oncopy` property was supported earlier (I confirmed with `document.body.oncopy` in IE8) I assume the event could be fired.

Reply:
Before version 9, Internet Explorer used an alternative name for `addEventListener`, "`attachEvent`".  I was just clarifying that I tested the event using the alt. name as well!  Factoring in the tests for event handlers as well to confirm they work, my test code looks something more like the following:

```html
<div id="test">
	<input id="source" width="100" value="Try copying text from this box..." />
	<input id="target" width="100" value="...and pasting it into this one" />
</div>

<script>
	try {
		var source = document.getElementById('source');
		var target = document.getElementById('target');

		source.oncopy = function() {
			alert('Copy (handler)!');
		};
		source.addEventListener('copy', function() {
			alert('Copy (event)!');
		});
		source.attachEvent('copy', function() {
			alert('Copy (event)!');
		});

		source.oncut = function() {
			alert('Cut (handler)!');
		};
		source.addEventListener('cut', function() {
			alert('Cut (event)!');
		});
		source.attachEvent('cut', function() {
			alert('Cut (event)!');
		});

		target.onpaste = function() {
			alert('Paste (handler)!');
		};
		target.addEventListener('paste', function() {
			alert('Paste (event)!');
		});
		target.attachEvent('paste', function() {
			alert('Paste (event)!');
		});
	} catch(e) {
		alert(e);
	}
</script>
```

It's weird.  While most events and event handlers are added at the same time, and those that aren't always had the event added first, this one seems to be the opposite, where the event handler came first...  I guess we should add a similar note?

Reply:
That is unusual indeed! I wonder if the events were fired on something other than `Element` before this? Maybe the event was fired on `Window` or `Document`, but the event handler properties were sprinkled around a bit too much?

Reply:
Hmm, the event seems to not fire on the window or document either.  I guess that it really is just catched with the event handler to begin with.  I'll add a note describing as such.

Reply:
Wait, so setting the `oncopy` property works that callback is invoked? If that but no other mechanism works, that's the first time I've seen that situation, bizarre!